timed/reports/management/commands/notify_reviewers_unverified.py

```diff
@@ -48,10 +48,24 @@ def add_arguments(self, parser):
             dest='offset',
             help='Period will end today minus given offset.'
         )
+        parser.add_argument(
+            '--message',
+            type=str,
+            dest='message',
+            help='Additional message to send if there are unverified reports'
+        )
+        parser.add_argument(
+            '--cc',
+            type=str,
+            dest='cc',
+            help='Email address where to send a cc'
+        )
 
     def handle(self, *args, **options):
         months = options['months']
         offset = options['offset']
+        message = options['message']
```

Comment:
not sure but what happens when I do not pass on `--message`. Will this be value then None which is then added to the email? I guess something to verify.

Reply:
I have set the default of message to an empty string.